.genx/prompts/api.js

```diff
@@ -7,19 +7,12 @@ const ssoIntro = () => console.log(TEXTS.INTRO_API_SSO);
 module.exports = async (inquirer, prevAns = {}) => {
   apiHostIntro();
   const { apiHost = prevAns.apiHost } = await inquirer.prompt([
-    {
-      name: 'setApiHost',
-      type: 'confirm',
-      message: TEXTS.MESSAGE_API_SET_HOST,
```

Comment:
Can get delete that `TEXTS.MESSAGE_API_SET_HOST` now

Reply:
AHH indeed! good one

Reply:
addressed in https://github.com/genesiscommunitysuccess/blank-app-seed/pull/343/commits/d54a0373aa21f0883c7d80b7f74dc586ccbcec3b